flang/test/Lower/format.f90

```diff
@@ -0,0 +1,30 @@
+! RUN: bbc %s -o - | FileCheck %s
+
+! CHECK-LABEL: func @_QPformatassign
+function formatAssign()
+    real :: pi
+    integer :: label
+    logical :: flag
+
+    ! CHECK: select
+    if (flag) then
+       assign 100 to label
+    else
+       assign 200 to label
+    end if
+
+    ! CHECK: fir.select
```

Comment:
Do you want to maybe expand this match more to make sure it has the expected arguments?